kf/kf_windows.go

```diff
@@ -55,3 +57,19 @@ func (b *BPF) ProcessTerminate() error {
 func VerifyNCreateTCDirs() error {
 	return nil
 }
+
+// AttachXDP - Attach BPF Program
+func AttachXDP(bpfRootProg *ebpf.Program, index int) error {
+	return fmt.Errorf("AttachXDP - UnSupported on windows")
+}
+
+// LoadTCRootProgram - not implemented in windows
+func (b *BPF) LoadTCRootProgram(ifaceName, direction string, eBPFProgram *BPF) error {
+	// not implement nothing todo
+	return fmt.Errorf("LoadTCRootProgram - TC programs UnSupported on windows")
+}
+
+// UnLoadTCProgram - Remove TC filters
+func (b *BPF) UnLoadTCProgram(ifaceName, direction string) error {
+	return fmt.Errorf("UnLoadTCProgram - TC programs UnSupported on windows")
```

Comment:
```suggestion
	return fmt.Errorf("UnloadTCProgram - TC programs Unsupported on windows")
```
nit: s/UnLoad/Unload/ throughout